main.js

```diff
@@ -129,6 +152,7 @@ async function main() {
       } else {
         alertAudio = null;
       }
+      Cookies.set('alertAudio', value, { expires: COOKIE_EXPIRATION_DAYS });
```

Comment:
Use Cookies.default to set the default cookies object so we don't need `{ expires: COOKIE_EXPIRATION_DAYS }` everywhere. Or set that set a variable like `const COOKIES_ATTR = { expires: COOKIE_EXPIRATION_DAYS }` This comment applies for other instances as well.